api/src/services/graphql/index.ts

```diff
@@ -1,112 +1,18 @@
-import { fetchAccountabilityPolicyGlobals } from '../../permissions/modules/fetch-accountability-policy-globals/fetch-accountability-policy-globals.js';
-import { fetchAllowedFieldMap } from '../../permissions/modules/fetch-allowed-field-map/fetch-allowed-field-map.js';
-import { fetchInconsistentFieldMap } from '../../permissions/modules/fetch-inconsistent-field-map/fetch-inconsistent-field-map.js';
-import { createDefaultAccountability } from '../../permissions/utils/create-default-accountability.js';
-import type { AbstractServiceOptions, AuthenticationMode, GraphQLParams } from '../../types/index.js';
-import { generateHash } from '../../utils/generate-hash.js';
-import { getGraphQLType } from '../../utils/get-graphql-type.js';
-import { getIPFromReq } from '../../utils/get-ip-from-req.js';
-import { getSecret } from '../../utils/get-secret.js';
+import type { AbstractServiceOptions, GraphQLParams } from '../../types/index.js';
 import { getService } from '../../utils/get-service.js';
-import isDirectusJWT from '../../utils/is-directus-jwt.js';
-import { verifyAccessJWT } from '../../utils/jwt.js';
-import { mergeVersionsRaw, mergeVersionsRecursive } from '../../utils/merge-version-data.js';
-import { reduceSchema } from '../../utils/reduce-schema.js';
-import { sanitizeQuery } from '../../utils/sanitize-query.js';
-import { validateQuery } from '../../utils/validate-query.js';
-import { AuthenticationService } from '../authentication.js';
-import { CollectionsService } from '../collections.js';
-import { ExtensionsService } from '../extensions.js';
-import { FieldsService } from '../fields.js';
-import { FilesService } from '../files.js';
-import { RelationsService } from '../relations.js';
-import { RevisionsService } from '../revisions.js';
-import { RolesService } from '../roles.js';
-import { ServerService } from '../server.js';
-import { SpecificationService } from '../specifications.js';
-import { TFAService } from '../tfa.js';
-import { UsersService } from '../users.js';
-import { UtilsService } from '../utils.js';
-import { VersionsService } from '../versions.js';
+import { formatError } from './errors/format.js';
 import { GraphQLExecutionError, GraphQLValidationError } from './errors/index.js';
-import { cache } from './schema-cache.js';
-import { createSubscriptionGenerator } from './subscription.js';
-import { GraphQLBigInt } from './types/bigint.js';
-import { GraphQLDate } from './types/date.js';
-import { GraphQLGeoJSON } from './types/geojson.js';
-import { GraphQLHash } from './types/hash.js';
-import { GraphQLStringOrFloat } from './types/string-or-float.js';
-import { GraphQLVoid } from './types/void.js';
+import { _getSchema } from './schema/index.js';
 import { addPathToValidationError } from './utils/add-path-to-validation-error.js';
 import processError from './utils/process-error.js';
-import { sanitizeGraphqlSchema } from './utils/sanitize-gql-schema.js';
+
+export type GQLScope = 'items' | 'system';
```

Comment:
Could we also put types and constants (like on line 28) that are exported into separate files?